paddle/fluid/framework/executor.cc

```diff
@@ -203,29 +206,36 @@ static bool has_feed_operators(
     if (op->Type() == kFeedOpType) {
       feed_count++;
       // The input variable's name of feed_op should be feed_holder_name.
-      PADDLE_ENFORCE_EQ(op->Input("X")[0], feed_holder_name,
-                        "Input to feed op should be '%s'", feed_holder_name);
+      PADDLE_ENFORCE_EQ(
+          op->Input("X")[0], feed_holder_name,
+          platform::errors::PreconditionNotMet(
+              "Input to feed op should be '%s'", feed_holder_name));
       std::string feed_target_name = op->Output("Out")[0];
-      PADDLE_ENFORCE(
-          feed_targets.find(feed_target_name) != feed_targets.end(),
-          "Feed operator output name '%s' cannot be found in 'feed_targets'",
-          feed_target_name);
+      PADDLE_ENFORCE_NE(feed_targets.find(feed_target_name), feed_targets.end(),
+                        platform::errors::PreconditionNotMet(
+                            "Feed operator output name '%s' cannot be found in "
+                            "'feed_targets'",
+                            feed_target_name));
     }
   }
 
   if (feed_count > 0) {
     PADDLE_ENFORCE_EQ(
         feed_count, feed_targets.size(),
-        "The number of feed operators should match 'feed_targets'");
+        platform::errors::PreconditionNotMet(
+            "The number of feed operators should match 'feed_targets'"));
```

Comment:
`"The number of feed operators should match 'feed_targets, but received feed_count: %d, required feed_targets.size():  %d.",  feed_count, feed_targets.size()` ?

Reply:
Done